src/Components/CurrentWeatherDisplay/CurrentWeatherDisplay.js

```diff
@@ -9,60 +9,54 @@ import './CurrentWeatherDisplay.css';
 import sun from './sun.png';
 import { countryCodes } from '../../constants/countrycodes';
 
-function Alert(props) {
+const Alert = (props) => {
   return <MuiAlert elevation={6} variant="filled" {...props} />;
-}
+};
 
-export class CurrentWeatherDisplay extends React.Component{
-  constructor(props){
-    super(props);
+export const CurrentWeatherDisplay = (props) => {
```

Comment:
Given that we're working with React, we should be using functional components (that is, no class) wherever possible since it's lightweight